File: 3790-fruits-into-baskets-ii/3790-fruits-into-baskets-ii.c
int numOfUnplacedFruits(int* fruits, int fruitsSize, int* baskets, int basketsSize) {
    int answer = 0;

    for (int i = 0; i < fruitsSize; i++){
        int fruit = fruits[i];
        int index = 0;
        int flag = 0;

        for (int j = 0; j < basketsSize; j++) {
            if (baskets[j] >= fruit) {
                baskets[j] = -1;
                flag = 1;
                break;
            }
        }

        if (!flag) {
            answer ++;
        }
    }
    return answer;
}

File: 3790-fruits-into-baskets-ii/3790-fruits-into-baskets-ii.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        
        answer = 0
        for fruit in fruits:
            index = 0
            while True:
                if index == len(baskets):
                    answer += 1
                    break
                if baskets[index] >= fruit:
                    del baskets[index]
                    break
                else:
                    index += 1
                
        return answer

            